easybuild/easyconfigs/l/libgit2/libgit2-1.7.2-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'libgit2'
+version = '1.7.2'
+
+homepage = 'https://libgit2.org/'
+description = """libgit2 is a portable, pure C implementation of the Git core methods provided as a re-entrant
+linkable library with a solid API, allowing you to write native speed custom Git applications in any language
+which supports C bindings."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = [GITHUB_SOURCE]
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['de384e29d7efc9330c6cdb126ebf88342b5025d920dcb7c645defad85195ea7f']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.27.6'),
+    ('pkgconf', '2.0.3'),
+]
+dependencies = [
+    ('PCRE2', '10.42'),
```

Comment:
This is a change from earlier ECs: What does libgit look for? `pcre` or `pcre2` or even neither?